example/README.md

```diff
@@ -0,0 +1,12 @@
+# Examples
+
+## How to Run
+
+This directory allows to check how some UI will look into various circumstances.
+
+As it use real systemd journal it have to be run as root. It is also recommended
```

Comment:
Strictly speaking, journalctl can also be run as normal user, although it will show very limited records.